packages/botonic-plugin-google-translate/README.md

```diff
@@ -0,0 +1,65 @@
+# Botonic Plugin Google Translate
+
+## What does this plugin do?
+This plugin uses [Google Cloud Translation API](https://cloud.google.com/translate) to detect the language of the user input text.
+
+## Setup
+
+### Install the plugin
+To install this plugin, you only need to execute the following command:
+
+```bash
+npm i @botonic/plugin-google-translate 
+```
+
+### Enable Cloud Translation API
+Before using this plugin, you need a project that has the Cloud Translation API enabled and its credentials. See [Cloud Translation Set Up Guide](https://cloud.google.com/translate/docs/setup).
+
+### Add the plugin
+
+To being able of using this plugin to detect the language of the user input text, you need to have it installed and added to your plugins.
```

Comment:
The sentence is verbose and does not add much meaning.
"You need to add the following configuration to your bot's plugins"